src/pages/[slug]/[ie]/index.tsx

```diff
@@ -636,6 +644,29 @@ const ObjectDetailPage: NextPage = () => {
 				onClose={onCloseBlade}
 				onSubmit={onCloseBlade}
 			/>
+		</>
+	);
+
+	const renderPageContent = () => {
+		if (mediaInfoIsLoading || visitRequestIsLoading) {
+			return <Loading fullscreen />;
+		}
+		if (isErrorNotFound) {
+			return <ErrorNotFound />;
+		}
+		if (isErrorSpaceNoAccess) {
+			return <ErrorSpaceNoAccess visitorSpaceSlug={router.query.slug as string} />;
+		}
```

Comment:
Waarom zijn dit geen redirects naar `404`- & `403`-pagina's?

Reply:
Dan verliest de gebruiker de url waar hij wilde naar toe gaan.
bv: hij wil een url in typen en maakt een typo.
Dan krijg je:
bezoek.hetarchief.be/huis-van-alin => bezoek.hetarchief.be/404

En dan moet de gebruiker die volledige opnieuw intypen